app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewsController < ApplicationController
+
+  def create
+    @card = Card.find(review_params[:card_id])
+    if @card.check_translation?(review_params[:input_text])
+      flash[:notice] = 'Правильный перевод'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.